src/components/data/data-utils.js

```diff
@@ -191,13 +186,12 @@ export const EntityDeleter = ({ onDismiss, onSuccess, namespace, name, selectedE
   }
 
   const total = selectedKeys.length + additionalDeletions.length
-  return h(Modal, {
-    onDismiss,
-    title: 'Confirm Delete',
-    okButton: h(ButtonPrimary, {
-      disabled: deleting,
-      onClick: doDelete
-    }, ['Delete'])
+  return h(DeleteConfirmationModal, {
```

Comment:
Per our conversation, consider whether this is providing the right amount of friction in each case.

Reply:
Removed the requirement to type a confirmation message from all the current confirmations that use this component (it still appears when deleting a workspace). Left it as an option since we may decide to use it when deleting an entire data table.